tasks/install-prometheus.yml

```diff
@@ -107,8 +107,16 @@
 - name: set prometheus variables
   copy: src="../files/etc-default-prometheus"  dest=/etc/default/prometheus
 
+- name: copy systemd config to server
+  template: src="../templates/prometheus.service.j2"  dest="/lib/systemd/system/prometheus.service"
```

Comment:
Why `/lib/systemd/system/prometheus.service` instead of `/etc/systemd/system/prometheus.service`?